pkg/backupdriver/backup_driver_controller_base.go

```diff
@@ -1049,11 +1049,11 @@ func (ctrl *backupDriverController) syncSvcSnapshotByKey(key string) error {
 	case backupdriverapi.SnapshotPhaseSnapshotFailed:
 		newSnapshotStatusPhase = svcSnapshot.Status.Phase
 	default:
-		ctrl.logger.Infof("syncUploadByKey: No change needed for snapshot phase %s", svcSnapshot.Status.Phase)
+		ctrl.logger.Debugf("syncSvcSnapshotByKey: No change needed for snapshot phase %s", svcSnapshot.Status.Phase)
 		return nil
 	}
 	snapshotStatusFields := make(map[string]interface{})
-	ctrl.logger.Infof("syncSvcSnapshotByKey: calling updateSnapshotStatusPhase %s/%s", snapshot.Namespace, snapshot.Name)
+	ctrl.logger.Debugf("syncSvcSnapshotByKey: calling updateSnapshotStatusPhase %s/%s", snapshot.Namespace, snapshot.Name)
```

Comment:
Please leave this line as it is. This line should be called constant times (depends on how many number of states of Snapshot CR) per CR.

Reply:
updateSnapshotStatusPhase logs it when it notices a phase change, I don't think it needs one more logging here.